### PR TITLE
New release, update eredis, add TCP option suggestions

### PR DESCRIPTION
**Pull request by zuiderkwast**
Update the eredis dependency.

Document suggested TCP options as discussed in #59.

New release 0.8.0.

### CODE REVIEW

**Comment by zuiderkwast** [comment]
@e-kmisk

File: README.md [hunk 1]
```diff
@@ -167,7 +170,32 @@ retrieve them through the command `CLUSTER SLOTS` at runtime.
...
+
+     ```Erlang
+     %% See Linux kernel file: include/uapi/linux/tcp.h
+     %%  - send keep-alive packets after 1s (minimum possible) of no traffic
+     %%  - time out after one failed try if the keep-alive is not acknowledged
+     %% TCP_KEEPIDLE in seconds - cannot be more granular here:
+     {raw, 6, 4, <<1:32/native>>},
+     %% TCP_KEEPCNT - try only once, fail fast, better than hanging:
+     {raw, 6, 6, <<1:32/native>>},
+     %% TCP_KEEPINTVL - probably does not matter when KEEPCNT == 1:
```
Comment (e-kmisk) [inline README.md:194]: > probably does not matter when KEEPCNT == 1

I was wrong in this, better remove that part of the comment.

File: README.md [hunk 2]
```diff
@@ -167,7 +170,32 @@ retrieve them through the command `CLUSTER SLOTS` at runtime.
 * `pool_size`: Number of connected clients to each Redis node. Default: `10`
 * `pool_max_overflow`: Max number of extra clients that can be started when the pool is exhausted. Default: `0`
 * `password`: Password to use for a Redis cluster configured with `requirepass`. Default: `""` (i.e. AUTH not sent)
-* `socket_options`: Extra socket [options](http://erlang.org/doc/man/gen_tcp.html#type-option). Enables selecting host interface or perf. tuning. Default: `[]`
+* `socket_options`: Extra socket [options](http://erlang.org/doc/man/gen_tcp.html#type-option). Enables selecting host interface or perf. tuning. Default: `[]`.
+   However, there are some options added by default by the underlaying eredis
```
Comment (e-kmisk) [inline README.md:174]: > underlaying 

underlying

File: README.md [hunk 3]
```diff
@@ -167,7 +170,32 @@ retrieve them through the command `CLUSTER SLOTS` at runtime.
...
+     errors are not ignored. Triggers a reconnect on send errors.
+   * `{nodelay, true}`: Send TCP packets immediately. A Redis command is sent as
+     a whole, not in pieces, so there is no point delaying the send.
+   * `{keepalive, true}`: Enables TCP keepalive. Enabled by default on Linux.
+   * Raw Linux-specific options, to detect a failing socket as soon as possible:
+
+     ```Erlang
+     %% See Linux kernel file: include/uapi/linux/tcp.h
+     %%  - send keep-alive packets after 1s (minimum possible) of no traffic
+     %%  - time out after one failed try if the keep-alive is not acknowledged
```
Comment (e-kmisk) [inline README.md:189]: >     %%  - send keep-alive packets after 1s (minimum possible) of no traffic
>     %%  - time out after one failed try if the keep-alive is not acknowledged

No need for these two lines, they are too specific as well I think.

File: README.md (hunk 1 above)
Comment (zuiderkwast) [reply README.md:194]: What should I write here instead? It's good to have some explanation of each raw options.

**Review by bjosv — APPROVED** [review]
LGTM, have no direct input regarding the linux specific socket options

File: README.md (hunk 1 above)
Comment (zuiderkwast) [reply README.md]: I used this, more or less exactly. Looks OK now?

**Comment by e-kmisk** [comment]
Hi,

Are you guys planning to add a tag or release entry for `0.8.0`? Atm. I cannot see either.

**Comment by zuiderkwast** [comment]
Yes, done. I had some trouble because I forgot about the Mix files and rebar.lock which needs to be deleted, otherwise the deps aren't actually updated. :-( I published it on hex and then I had to do it again (on hex you have an hour to fix trouble with a new release) and after that I tagged it and pushed the tag.

**Pull request merged**